src/CovidSim.cpp

```diff
@@ -520,8 +520,8 @@ void ReadParams(char* ParamFile, char* PreParamFile)
 	if (P.DoAdUnits)
 	{
 		char** AdunitNames, * AdunitNamesBuf;
-		if (!(AdunitNames = (char**)malloc(3 * ADUNIT_LOOKUP_SIZE * sizeof(char*)))) ERR_CRITICAL("Unable to allocate temp storage\n");
-		if (!(AdunitNamesBuf = (char*)malloc(3 * ADUNIT_LOOKUP_SIZE * 360 * sizeof(char)))) ERR_CRITICAL("Unable to allocate temp storage\n");
+		if (!(AdunitNames = (char**)calloc(3, ADUNIT_LOOKUP_SIZE * sizeof(char*)))) ERR_CRITICAL("Unable to allocate temp storage\n");
+		if (!(AdunitNamesBuf = (char*)calloc(3,  ADUNIT_LOOKUP_SIZE * 360 * sizeof(char)))) ERR_CRITICAL("Unable to allocate temp storage\n");
```

Comment:
Can we be consistent with other places here please - and so the second argument should just be a `sizeof`?

```suggestion
		if (!(AdunitNames = (char**)calloc(3 * ADUNIT_LOOKUP_SIZE, sizeof(char*)))) ERR_CRITICAL("Unable to allocate temp storage\n");
		if (!(AdunitNamesBuf = (char*)calloc(3 * ADUNIT_LOOKUP_SIZE * 360, sizeof(char)))) ERR_CRITICAL("Unable to allocate temp storage\n");
```